### DISCUSSION TITLE
feat: add drawer or allow panel to be used inside a container

### DISCUSSION THREAD

**Discussion by rfrancisco** [microsoft/fluentui-blazor#1463] [Ideas]
# 🙋 Feature Request

The panel component is useful but it's limited in its uses since it cannot be hosted inside a parent container. Having a drawer component, like the one in the react implementation of fluentui would be great. 

## 🤔 Expected Behavior

Host a panel like component inside the main content area or other containers

## 💻 Examples

Any drawer implementation like the ones in the react implementation of [flueentui](https://react.fluentui.dev/?path=/docs/components-drawer--default)

**Comment by dvoituron** [comment]
Hi,
Indeed, this could be an interesting improvement. But it is currently not yet planned in FluentUI Blazor. maybe for a next major release. Or if you'd like to create a PR with this feature, you're welcome :-)

**Comment by vnbaaij** [comment]
Moving this to discussion to gauge interest
